ospfd/ospf_dump.c

```diff
@@ -1447,6 +1449,33 @@ DEFUN (no_debug_ospf_sr,
 	return CMD_SUCCESS;
 }
 
+DEFUN (debug_ospf_default_info,
+       debug_ospf_default_info_cmd,
+       "debug ospf default info",
+       DEBUG_STR
+       OSPF_STR
+       "OSPF default information\n")
```

Comment:
Missing help string for "info". Maybe it's better to use "default-info" instead.

Reply:
Done.